cmd/esbuild/main.go

```diff
@@ -300,7 +300,7 @@ func main() {
 			for _, arg := range osArgs {
 				if !strings.HasPrefix(arg, "-") {
 					nonFlagCount++
-				} else if arg == "--serve" || arg == "--watch" || strings.HasPrefix(arg, "--serve=") {
+				} else if arg == "--watch" || strings.HasPrefix(arg, "--watch=") || arg == "--serve" || strings.HasPrefix(arg, "--serve=") {
```

Comment:
This seems inaccurate for `=false` options (it already was for `serve`, now also for `watch`)